engines/sci/graphics/paint32.cpp

```diff
@@ -113,24 +112,26 @@ void GfxPaint32::plotter(int x, int y, int color, void *data) {
 			properties.patternIndex = 0;
 		}
 	} else {
-		warning("GfxPaint32::plotter: Attempted to write out of bounds (%u >= %u)", index, properties.bitmap->getDataSize());
+		warning("GfxPaint32::plotter: Attempted to write out of bounds (%u, %u) - bounds are (%u, %u)", x, y, properties.bitmap->getWidth(), properties.bitmap->getHeight());
```

Comment:
This warns a lot; I wonder if there is another defect somewhere else causing overdraw.